app/vmalert/notifier/init_test.go

```diff
@@ -35,3 +36,63 @@ func TestInit(t *testing.T) {
 		t.Fatalf("expected to get \"127.0.0.2/api/v2/alerts\"; got %q instead", nf2.Addr())
 	}
 }
+
+func TestInitBlackHole(t *testing.T) {
+
+	*blackHole = true
```

Comment:
flags are global variables. If tests need to modify them, then you need to rollback the change once test is completed. This is related to all tests in this PR.

Reply:
will modify